src/test/java/com/timgroup/statsd/UnixStreamSocketTest.java

```diff
@@ -159,16 +161,19 @@ public void resist_dsd_timeout() throws Exception {
 
         // Freeze the server to simulate dsd being overwhelmed
         server.freeze();
-        while(lastException.getMessage() == null) {
+
+        while (lastException.getMessage() == null) {
             client.gauge("mycount", 20);
             Thread.sleep(10);  // We need to fill the buffer, setting a shorter sleep
         }
-        String excMessage = isLinux() ? "Resource temporarily unavailable" : "No buffer space available";
+        String excMessage = "Write timed out";
         assertThat(lastException.getMessage(), containsString(excMessage));
 
         // Make sure we recover after we resume listening
         server.clear();
         server.unfreeze();
+
+        // Now make sure we can receive gauges with 30
```

Comment:
I get consistent timeouts on this test when I run the tests locally (intel mac, jdk8)

```
resist_dsd_timeout(com.timgroup.statsd.UnixStreamSocketTest)  Time elapsed: 10.025 sec  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 10000 milliseconds
	at com.timgroup.statsd.UnixStreamSocketTest.resist_dsd_timeout(UnixStreamSocketTest.java:167)
```